src/main/java/com/faforever/client/chat/UserFilterController.java

```diff
@@ -33,6 +36,8 @@
   public TextField clanFilterField;
   public TextField minRatingFilterField;
   public TextField maxRatingFilterField;
+  public ToggleGroup gameStatusToggleGroup;
+  private BooleanProperty isFilterApplied = new SimpleBooleanProperty(false);
```

Comment:
* As a convention, we always initialize fields in the constructor, never in the declaration
* Since this field should never change, it should be `final`
* The JavaBean naming convention for boolean fields is `isXXX` where `XXX` is the property name. Therefore, the prefix `is` in the field should be removed, otherwise we'll get `isIsFilterApplied`